gammapy/spectrum/models.py

```diff
@@ -8,6 +8,13 @@
 from ..extern.bunch import Bunch
 from ..utils.energy import EnergyBounds
 
+# This cannot be made a delayed import because the pytest matrix fails if it is
```

Comment:
Can you leave a TODO here, with a link to the relevant travis-ci log and maybe this PR.
I'd like to try and reproduce this locally later and understand it.

Something is very wrong either with global state in Sherpa or Gammapy or circular imports between `models.py` and `sherpa_models.py`.